src/selfplay/tournament.cc

```diff
@@ -51,6 +52,7 @@ void SelfPlayTournament::PopulateOptions(OptionsParser* options) {
   options->AddContext("player2");
 
   options->Add<BoolOption>(kShareTreesStr, "share-trees") = false;
+  options->Add<BoolOption>(kNoTreeReuseStr, "disable-tree-reuse") = false;
```

Comment:
It's better to replace no-tree-reuse=false  to reuse-tree=true.
(make this parameter mean the opposite, with opposite default.  But actually disabling tree reuse by default is probably right thing to do).

Also the same for variables. not no_tree_reuse, but reuse_tree

Reply:
Done.